docs/reference/attacks/index.md

```diff
@@ -30,6 +30,6 @@ hide:
 | [TOKEN_BRUTEFORCE](./TOKEN_BRUTEFORCE.md) | Brute-force secret name of service account token | Steal Application Access Token | Credential Access | 
 | [TOKEN_LIST](./TOKEN_LIST.md) | Access service account token secrets | Steal Application Access Token | Credential Access | 
 | [TOKEN_STEAL](./TOKEN_STEAL.md) | Steal service account token from volume | Unsecured Credentials | Credential Access | 
-| [TOKEN_VAR_LOG_SYMLINK](./TOKEN_VAR_LOG_SYMLINK.md) | Steal service account token from volume | Unsecured Credentials | Credential Access | 
+| [CE_VAR_LOG_SYMLINK](./CE_VAR_LOG_SYMLINK.md) | Steal service account token from volume | Unsecured Credentials | Credential Access | 
```

Comment:
I think the MITRE also needs to change to match EXPLOIT_HOST_READ

Reply:
good catch, using the same as:
```
| [EXPLOIT_HOST_READ](./EXPLOIT_HOST_READ.md) | Read file from sensitive host mount | Escape to host | Privilege escalation | 
```